students/adam_strong/lesson07/html_render.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element(object):
+    tag = 'html'
+    indent = '    '
+    indlvl = 0
+
+    def __init__(self, content=None, **kwargs):
+        self.content = []
+        self.kwargs = kwargs
+        if content is not None:
+            self.content = [content,'\n']
+
+
+    def append(self, content):
+        self.content.append(content)
+
+
+    def front_tag(self):
+        k = ''
+        for key in self.kwargs:
+            k = k + ' ' + key + "='" + str(self.kwargs[key]) + "'"
+        self.content.insert(0,'<' + self.tag + k + '>')
+
+
+    def back_tag(self):
+        self.content.insert(len(self.content),'</' + self.tag + '>')
+
+
+    def render(self, out_file, cur_ind=''):
+        self.cur_ind = cur_ind + (self.indent*self.indlvl)
+        self.front_tag()
+        self.content.insert(1,'\n')
+        self.back_tag()
+        self.content.insert(len(self.content),'\n')   
+        #Probably going to have to add indent
+        for contentlet in self.content:
+            try:
+                contentlet.render(out_file)
+            except AttributeError:
+                if contentlet is '\n':
```

Comment:
You'll want to us == in this instance, as well as throughout the rest of this code